build.gradle

```diff
@@ -325,6 +336,12 @@ dependencies {
     // Required for SV Discovery machine learning
     compile group: 'biz.k11i', name: 'xgboost-predictor', version: '0.3.0'
 
+    testUtilsCompile sourceSets.main.output
+    testUtilsCompile 'org.testng:testng:' + testNGVersion //compile instead of testCompile because it is needed for test infrastructure that needs to be packaged
+    testUtilsCompile 'org.apache.hadoop:hadoop-minicluster:' + hadoopVersion
+
+    testCompile sourceSets.testUtils.output
+
     testCompile "org.mockito:mockito-core:2.10.0"
```

Comment:
Can/should mockito be moved to testUtilsCompile ?

Reply:
It can, but there's no reason to do so.  Nothing in testUtils uses it.  We can move it when / if we need it.  Best to keep dependencies scoped as narrowly as possible I think.  Similar issue with jimfs.  I suspect your ArgumentsBuilder branch will require jimfs to move to testUtils.